analytics/services/submission_analytics.py

```diff
@@ -1,22 +1,229 @@
+from accounts.models import MyUser
```

Comment:
This file is too big. You have to split it into multiple files.

Reply:
I removed a lot of redundant code, should be better now.